modules/core/src/passes/layers-pass.js

```diff
@@ -218,13 +218,18 @@ export default class LayersPass extends Pass {
 export function layerIndexResolver(startIndex = 0, layerIndices = {}) {
   const resolvers = {};
 
-  return (layer, isDrawn) => {
+  const resolveLayerIndex = (layer, isDrawn) => {
     const indexOverride = layer.props._offset;
     const layerId = layer.id;
     const parentId = layer.parent && layer.parent.id;
 
     let index;
 
+    if (parentId && !(parentId in layerIndices)) {
+      // Populate layerIndices with the parent layer's index
```

Comment:
If I was picky, I'd say this comment is perhaps this is a bit abstract for our hypothetical successors, a bit more comment on the context where this happens when rendering tiled layers etc might make it a bit easier to understand why this happens.

In general the comment header on top of this function may need a pass, e.g. it talks about an `_index` prop which is not referenced in the code.